app/subscriber/src/components/basic-search/BasicSearch.tsx

```diff
@@ -77,8 +77,8 @@ export const BasicSearch = ({ onSearch, inHeader }: IBasicSearchProps) => {
           Search
           <FaPlay />
         </Button>
+        <p onClick={() => navigate(`/search/advanced/${filterId}`)}>GO ADVANCED</p>
```

Comment:
not your code originally I know, but a `<p` tag acting as a link is a bit funny.  Maybe it should be a `<link` instead?